frontend/src/components/admin-page/components/types/prop-function-void.type.ts

```diff
@@ -0,0 +1,2 @@
+type PropFunctionType<T> = (value:T)=> void
```

Comment:
`components` folder should include only components
move it to the `frontend/src/components/admin-page/common/types`